### DISCUSSION TITLE
Viewing the DBT process as a waterfall

### DISCUSSION THREAD

**Discussion by nikitok** [dagster-io/dagster#24217] [Q&A] [answered]
is it possible to view the execution schedule of a group of assets as a waterflow? When I launch a group for materialization, in the "Runs" tab, I see only 1 progress bar, but it would be interesting to see how long each step (asset) is executed for further optimization.

There is example of running 16 assets in dbt.
![dbt_asset_in_waterfall_mode](https://github.com/user-attachments/assets/504565a7-1e54-41d8-b772-bc8596ee5b96)

**Comment by garethbrickman** [comment, marked as answer]
This button toggles the `Waterfall view` for the steps:
![Screenshot 2024-09-04 at 16 40 11](https://github.com/user-attachments/assets/181074c9-10a0-40f2-a602-b9537bdb8287)

But as this dbt operation is a single step it will not break it down by each dbt asset materialized. Please feel free to post [a feature request](https://github.com/dagster-io/dagster/issues/new?assignees=&labels=type%3A+feature-request&projects=&template=request_a_feature.yml) for that.
